core/templates/dev/head/pages/exploration-player-page/layout-directives/learner-view-info.directive.ts

```diff
@@ -35,24 +37,30 @@ angular.module('oppia').directive('learnerViewInfo', [
     return {
       restrict: 'E',
       scope: {},
-      bindToController: {
-        explorationTitle: '@'
```

Comment:
Why we have deleted this binding?

Reply:
We are now fetching the exploration title via ajax